lib/modules/styleTweaks.js

```diff
@@ -111,6 +111,12 @@ modules['styleTweaks'] = {
 			description: 'Add an icon in the omnibar (where the page address is written) to disable/enable current subreddit style.',
 			noconfig: true // only show for chrome
 		},
+		disableSubredditStyle: {
+			type: 'boolean',
+			value: false,
+			description: 'Always remove subreddit style.'
```

Comment:
probably yeah. it would pair well with #945. also #963.  

could do something like how userbar collapser notifies you on next page load "uh are you sure you wanted to do that?"  but for this case, since it's a setting, it'll need a little more architecture.. which I'd rather defer until I hear what kinds of questions/confusions people get into.